### ISSUE TITLE
Unrecognized arguments

### ISSUE THREAD

**Issue by AlbertSmit** [GoogleCloudPlatform/github-actions#156]
### Question
Hey all!

I'm trying to run cache invalidation from a Github Action, but it does not seem to work.
After setting up **gcloud**, I run the following:
``` yaml   
- name: Invalidate cloud load balancer cache
  run: |-
    gcloud compute url-maps invalidate-cdn-cache redacted-url-map --path /* --async
```
_(as per https://cloud.google.com/sdk/gcloud/reference/compute/url-maps/invalidate-cdn-cache)._

When the workflow runs, it gives the following error:
``` bash
ERROR: (gcloud.compute.url-maps.invalidate-cdn-cache) unrecognized arguments:
  /boot
  /data
  /dev
  /etc
  /home
  /imagegeneration
  /lib
  /lib32
  /lib64
  /lost+found
  /media
  /mnt
  /opt
  /proc
  /root
  /run
  /sbin
```

Debugging
---
First off, I ran `gcloud config list`.
This outputs my details as expected.

Secondly, I did try to list my `redacted-url-map`, by running a job with the following:
```gcloud compute url-maps list```
This outputs the actual map I am trying to use, and confirms that I can use `gcloud compute`(?).

TL;DR
---
It seems to me that running `invalidate-cdn-cache` is not supported through `gcloud` in Github Actions; is that a correct assumption? Or am I not seeing / missing something?

Thanks alot!

**Comment by averikitsch** [comment]
Hi @AlbertSmit, `gcloud compute url-maps invalidate-cdn-cache redacted-url-map --path /* --async` doesn't run locally either. Adding quotes around the path fixes the problem both locally and on Github Actions. 

```
gcloud compute url-maps invalidate-cdn-cache redacted-url-map --path '/*' --async
```
